src/cargo-kani/src/args.rs

```diff
@@ -0,0 +1,313 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use anyhow::bail;
+use clap::{arg_enum, Error, ErrorKind};
+use std::ffi::OsString;
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+#[structopt(
```

Comment:
As discussed offline, we should plan to migrate out of structopt since it is now in maintenance mode: https://docs.rs/structopt/latest/structopt.